mongo/integration/unified_spec_test.go

```diff
@@ -480,23 +484,29 @@ func executeTestRunnerOperation(mt *mtest.T, testCase *testCase, op *operation,
 		mt.TrackFailPoint(fp.ConfigureFailPoint)
 	case "configureFailPoint":
 		fp, err := op.Arguments.LookupErr("failPoint")
-		assert.Nil(mt, err, "failPoint not found in arguments")
+		if err != nil {
+			return fmt.Errorf("unable to find 'failPoint' in arguments: %v", err)
```

Comment:
```suggestion
			return fmt.Errorf("unable to find 'failPoint' in arguments: %w", err)
```

(**) Wrapping errors is best practice: https://go.dev/blog/go1.13-errors#wrapping-errors-with-w

Reply:
Done